.storybook/preview-head.html

```diff
@@ -1,4 +1,4 @@
-<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:100,400,700,900">
+<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700">
```

Comment:
@JPorry in #15 , you mentioned that we use 300, 500, and 700. I think that 500 refers to the semibold weight, whereas 400 is the "normal" weight.

Typically, in my apps I use normal (400) most of the time, and then 500 for "bold" things. I think it'd be cool to include both 400 and 500 at least. I can't think of instances where I use 700, but I left it in here.

What do you think?

Reply:
yeah, makes sense. 300,400,500 sounds like enough.